packages/ember-runtime/lib/mixins/sortable.js

```diff
@@ -21,6 +21,8 @@ import {
   observer
 } from "ember-metal/mixin"; //ES6TODO: should we access these directly from their package or from how their exposed in ember-metal?
 
+export var sortableMixinDeprecation = 'Ember.SortableMixin is deprecated. Please use Ember.computed.sort instead.';
```

Comment:
Can you update the message to include the `this` value, without that the message doesn't really help you track it down without manually putting a breakpoint somewhere.